Buddies/Activities/ActivityCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ActivityCell.swift
+//  Buddies
+//
+//  Created by Grant Yurisic on 2/13/19.
+//  Copyright © 2019 Jack and the Beans. All rights reserved.
+//
+
+import UIKit
+
+class ActivityCell: UITableViewCell {
+
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var dateLabel: UILabel!
+    
+    @IBOutlet weak var descriptionLabel: UILabel!
+    
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var extraPicturesLabel: UILabel!
+    @IBOutlet weak var picture1: UIButton!
+    
+    @IBOutlet weak var picture2: UIButton!
```

Comment:
Do you think it'd be better to do an array of pics? I'd imagine we might have some activities with more or less ppl

Reply:
idk if there's some way we could handle it dynamically, ya know

Reply:
[Something to look into](https://stackoverflow.com/questions/24052459/swift-iboutletcollection-equivalent)